feat(config): add Yale Assure 2 Biometric Deadbolt locks

oandrew opened the pull request:
<!--
  Did you know? 🥳

  We now have preconfigured online instances of VSCode that help you through the contributing process
  without having to download and install a bunch of stuff on your system.
  These have auto-formatting and let you run checks, so prefer using them over editing config files on Github.

  https://gitpod.io/#/https://github.com/zwave-js/node-zwave-js
-->

Add configs for Yale Assure 2 Biometric Deadbolt locks:
```
YRD410-F-ZW3 Keyed Push Button Biometric Deadbolt
YRD420-F-ZW3 Keyed Touch Screen Biometric Deadbolt
YRD430-F-ZW3 Keyless Push Button Biometric Deadbolt
YRD450-F-ZW3 Keyless Touch Screen Biometric Deadbolt
```

Product IDs / Configuration IDs based on the official manual: https://products.z-wavealliance.org/ProductManual/File?folder=&filename=cert_portal/certs/859/YRD420_F_ZW3_MNL_1.pdf

Use a single device config file since they are functionally identical and differ only by keyed/keyless and push button/touch screen options.

Tested with my own `YRD420-F-ZW3`

Closes: #6611
Adds most of the locks from #6503

oandrew commented:
@AlCalzone  PTAL when you have a chance

AlCalzone commented on packages/config/config/devices/0x0129/yrd4x0-f-zw3.json:
```diff
@@ -0,0 +1,87 @@
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "templates/yale_template.json#volume_two_options"
```
According to the manual, this has 3 options

oandrew replied:
Fixed + added correct `defaultValue`

AlCalzone commented on packages/config/config/devices/0x0129/yrd4x0-f-zw3.json:
```diff
@@ -0,0 +1,87 @@
+		},
+		{
+			"#": "4",
+			"$import": "templates/yale_template.json#wrong_code_limit_3_to_10"
```
According to the manual, the default value is 5

oandrew replied:
Current default was 5 but according to the manual the default is 3. Fixed

AlCalzone commented on packages/config/config/devices/0x0129/yrd4x0-f-zw3.json:
```diff
@@ -0,0 +1,87 @@
+			"$import": "templates/yale_template.json#one_touch",
+			"defaultValue": 255
+		},
+		{
+			"#": "12",
+			"$import": "templates/yale_template.json#privacy_button"
+		},
```
This one is not documented in the manual

oandrew replied:
Removed

AlCalzone commented on packages/config/config/devices/0x0129/yrd4x0-f-zw3.json:
```diff
@@ -0,0 +1,87 @@
+			"#": "13",
+			"$import": "templates/yale_template.json#lock_status_led"
+		},
+		{
```
Param 18 is missing here, but documented.

oandrew replied:
Added it + fixed incorrect `minValue` in the `yale_template.json`

AlCalzone commented on packages/config/config/devices/0x0129/yrd4x0-f-zw3.json:
```diff
@@ -0,0 +1,87 @@
+		},
+		{
+			"#": "19",
+			"$import": "templates/yale_template.json#dps_alarm"
```
Default is enabled

oandrew replied:
Fixed

AlCalzone reviewed (changes requested):
I'm not entirely sure if we should include the actual device labels (this can be done conditionally), split the file into 4, or keep it as is. Other than that:

oandrew commented:
> I'm not entirely sure if we should include the actual device labels (this can be done conditionally), split the file into 4, or keep it as is.

Updated to use conditional label based on `productId`

p.s. 
IMO ideally zwave-js would support  per-device `label` / `description` properties to simplify this. e.g.
```
"devices": [
		//  (2nd Generation Assure Keyed Push Button Biometric Deadbolt)
		{
			"productType": "0x8107",
			"productId": "0x49d1",
			"zwaveAllianceId": 4936,
                        "label": "YRD410-F-ZW3",
                        "description": "Assure 2 Keyed Push Button Biometric Deadbolt"
		},
```
